docs/en/setup/backend/backend-storage.md

```diff
@@ -294,7 +294,7 @@ storage:
     username: ${SW_STORAGE_IOTDB_USERNAME:root}
     password: ${SW_STORAGE_IOTDB_PASSWORD:root}
     storageGroup: ${SW_STORAGE_IOTDB_STORAGE_GROUP:root.skywalking}
-    sessionPoolSize: ${SW_STORAGE_IOTDB_SESSIONPOOL_SIZE:0} # Zero means the size is 2*CPU_Cores
+    sessionPoolSize: ${SW_STORAGE_IOTDB_SESSIONPOOL_SIZE:8} # Zero means the size is 2*CPU_Cores
```

Comment:
Please adjust the comments to follow the codes.

Reply:
OK